### PR TITLE
Add SBLGNT trees

### PR DESCRIPTION
**Pull request by ryderwishart**
## Overview
This PR adds a version of the `macula-greek` syntax trees based on the SBLGNT.

@RickBrannan and @ryderwishart did the bulk of the work to get the Nodes and Lowfat versions of the syntax trees built.  @jacobwegner added TSV and TEI exports that mimic what was already offered for the Nestle1904 version of the trees (via #80, #81, #84)

## Subtasks
- [x] Nodes trees in macula-greek format (without head word attributes bubbled up to ancestor wgs
- [x] Lowfat trees in macula-greek format
- [x] Ensure SBLGNT trees do not exhibit any data loss (aside from textual variants) when compared with N1904 trees
- [x] Make sure SBLGNT lowfat attributes are named appropriately
- [x] Add `sblgnt/tei` for compatability with the existing Nestle1904 data (@jacobwegner)
- [x] Remove extra TSV href files if we don't want those

### CODE REVIEW

File: SBLGNT/nodes/23-1john.xml [hunk 1]
```diff
@@ -9374,7 +9374,7 @@
...
+            <Node Id="1.63" Cat="adv" Start="19" End="19" Lemma="kai/" StrongNumber="2532" English="also" Notes="" UnicodeLemma="καί" StrongNumberX="2532" Unicode="καὶ" Chinese="" Label="" Language="G" morphId="62005001020" encode="καὶ" RefIndex="" MorphId2="" RB="{}" NB="{}" AR="{}" BR="{}" nodeId="620050010200010" xml:id="n62005001020" ref="1JN 5:1!20" NormalizedForm="καί" UnicodeForm="καὶ" status="unmapped">καὶ</Node>
           </Node>
           <Node Id="1.64" Cat="np" Start="20" End="23" Rule="DetCL" Head="1" nodeId="620050010210040" Length="4">
             <Node Id="1.65" Cat="det" Start="20" End="20" StrongNumber="3588" English="the one" Case="Accusative" UnicodeLemma="ὁ" Unicode="τὸν" Chinese="（者）" Gender="Masculine" Number="Singular" Language="G" morphId="62005001021" encode="τὸν" nodeId="620050010210010" xml:id="n62005001021" ref="1JN 5:1!21" NormalizedForm="τόν" UnicodeForm="τὸν" FunctionalTag="T-ASM" Type="" FormalTag="T-ASM" Gloss="the [one]" LexDomain="092004" LN="92.24" status="mapped">τὸν</Node>
```
Comment (ryderwishart) [inline SBLGNT/nodes/23-1john.xml]: @RickBrannan, looking good in general. Thank you! Are "unmapped" words the only ones that have the additional values as with the preceding sibling KAI in this verse?

Comment (ryderwishart) [reply SBLGNT/nodes/23-1john.xml]: "Additional values" including things like `@Lemma`, which I see in ` <Node Id="1.63"...` but not  <Node Id="1.64"...`.

Comment (RickBrannan) [reply SBLGNT/nodes/23-1john.xml]: @ryderwishart , Words in SBLGNT that I wasn't able to locate in N1904 using mapping data have the `status="unmapped` attribute. _Some_ of those `unmapped` items, however, did have data that could be migrated to `Ref`, `SubjRef`, and `Frame` attributes. So where possible I've migrated that data. 
In those `unmappped` nodes, the following attributes (if present) have been removed: 'Lemma', 'Label', 'Notes', 'StrongNumberX', 'RB', 'NB', 'AR', 'BR', 'MorphId2','RefIndex', 'Ref', 'SubjRef', 'Frame', 'RefEnglish', 'RefStrong', 'RefLemma','SubjRefEnglish', 'SubjRefStrong', 'SubjRefLemma'
What I'm unaware of in those cases is what required data, if any, need be added to the SBLGNT edition.

Comment (RickBrannan) [reply SBLGNT/nodes/23-1john.xml]: There are `FormalTag` and `FunctionalTag` attributes in the N1904 that, for unmappable nodes, will not be available in SBLGNT. Should I try to get that data in (only `FormalTag`, I think, unsure how I'd get the `FunctionalTag` information) either based on the attributes that record morph fields or from Tauber's MorphGNT?

**Comment by jacobwegner** [comment]
@ryderwishart I added a `TODO` above for the TEI.

Were you thinking of generating a TSV as well at some point?

**Comment by ryderwishart** [comment]
> Were you thinking of generating a TSV as well at some point?

@jacobwegner Yes, I was planning on regenerating the TSV after the lowfat, as the TSV currently comes from the lowfat trees (IIRC).

**Pull request merged**